src/shared/format-table.h

```diff
@@ -102,7 +109,7 @@ int table_add_many_internal(Table *t, TableDataType first_type, ...);
 void table_set_header(Table *table, bool b);
 void table_set_width(Table *t, size_t width);
 void table_set_cell_height_max(Table *t, size_t height);
-int table_set_empty_string(Table *t, const char *empty);
+void table_set_replacement_string(Table *t, TableErsatz replacement);
```

Comment:
hmm, why does the function clal this "replacement", and the type "ersatz"? Stcik to one or the other, but not both?

Reply:
OK, which one do you prefer? I'd go for "ersatz" because it's shorter, but maybe people find it strange.

Reply:
OK, updated with "ersatz".